src/Backoffice/Task/App/Requests/StoreTaskRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Task\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use Lightit\Backoffice\Employee\Domain\Models\Employee;
+use Lightit\Backoffice\Task\Domain\DataTransferObjects\TaskDto;
+use Lightit\Backoffice\Task\Domain\Enums\TaskStatus;
+use Lightit\Backoffice\Task\Domain\Models\Task;
+
+class StoreTaskRequest extends FormRequest
+{
+    public const TITLE = 'title';
+
+    public const DESCRIPTION = 'description';
+
+    public const STATUS = 'status';
+
+    public const EMPLOYEE_ID = 'employee_id';
```

Comment:
FE should always send and receive camelCase

```suggestion
    public const EMPLOYEE_ID = 'employeeId';
```